### PR TITLE
Underscore unused arguments

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: lib/blacklight/solr/search_builder_behavior.rb [hunk 1]
```diff
@@ -112,7 +112,7 @@ def add_facetting_to_solr(solr_parameters)
         if facet.pivot
           solr_parameters.append_facet_pivot with_ex_local_param(facet.ex, facet.pivot.join(","))
         elsif facet.query
-          solr_parameters.append_facet_query facet.query.map { |k, x| with_ex_local_param(facet.ex, x[:fq]) }
+          solr_parameters.append_facet_query facet.query.map { |_k, x| with_ex_local_param(facet.ex, x[:fq]) }
```
Comment (cbeer) [inline lib/blacklight/solr/search_builder_behavior.rb]: I wonder if using e.g. `values.map` would be clearer? :shrug:

File: lib/blacklight/engine.rb [hunk 2]
```diff
@@ -8,7 +8,7 @@ class Engine < Rails::Engine
 
     # BlacklightHelper is needed by all helpers, so we inject it
     # into action view base here. 
-    initializer 'blacklight.helpers' do |app|
+    initializer 'blacklight.helpers' do |_app|
```
Comment (cbeer) [inline lib/blacklight/engine.rb]: Drop these unused block arguments entirely?

File: app/helpers/blacklight/facets_helper_behavior.rb [hunk 3]
```diff
@@ -8,7 +8,7 @@ module Blacklight::FacetsHelperBehavior
   # @param [Array<String>] fields
   # @param [Hash] options
   # @return [Boolean]
-  def has_facet_values? fields = facet_field_names, options = {}
```
Comment (cbeer) [inline app/helpers/blacklight/facets_helper_behavior.rb]: I don't know how downstream applications would use `options` here; maybe we can just drop it as part of this major release?

File: lib/railties/blacklight.rake [hunk 4]
```diff
@@ -24,7 +24,7 @@ namespace :blacklight do
 
   namespace :check do
     desc "Check the Solr connection and controller configuration"
-    task :solr, [:controller_name] => [:environment] do |_, args|
+    task :solr, [:controller_name] => [:environment] do |_, _args|
```
Comment (cbeer) [inline lib/railties/blacklight.rake]: Drop the unused block arguments?

**Comment by jcoyne** [comment]
@cbeer Changes made.

**Pull request merged**